radio-service-api/src/main/java/com/pnu/dev/radioserviceapi/client/dto/search/YoutubeSearchResponse.java

```diff
@@ -1,10 +1,11 @@
 package com.pnu.dev.radioserviceapi.client.dto.search;
 
+import lombok.Builder;
 import lombok.Data;
 
 import java.util.List;
 
-
+@Builder(toBuilder = true)
```

Comment:
do you use toBuilder() for all the classes, where you added it?

Reply:
Any of these classes do not have constructors or setters so field initialization is impossible. That is why I decided to use _toBuilder_